index.js

```diff
@@ -93,69 +84,70 @@ async function ensureRepoIsAccessible(repo) {
 }
 
 async function init() {
-	await verifyToken();
-	const query = new URLSearchParams(location.search);
-	let match;
+	await waitForToken();
+
+	let repo;
+	let ref;
+	let dir;
+
 	try {
+		const query = new URLSearchParams(location.search);
 		const parsedUrl = new URL(query.get('url'));
-		match = repoDirRegex.exec(parsedUrl.pathname);
-		if (!match) {
-			return updateStatus();
-		}
+		[, repo, ref, dir] = repoDirRegex.exec(parsedUrl.pathname);
+
+		console.log('Source:', {repo, ref, dir});
 	} catch (_) {
```

Comment:
I'd like to use optional catch binding, like `} catch {`. However, Parcel currently does not transform that due to the "latest 1 Chrome version" browser setting.

Now that the project gets a URL input and becomes kind of usable standalone, we may want to add other modern browsers to that list.

Reply:
Good _catch_ on this. Honestly I don't care about other browsers, this tool is not for IE users and we're not using anything bleeding edge.

Both browsers have supported this feature for _the last 12 versions_ and even Safari since 2018. I'll add it as is